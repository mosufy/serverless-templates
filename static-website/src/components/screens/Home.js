import React from "react";

import HomeHeader from "Components/modules/HomeHeader";

const Home = () => {
  return <HomeHeader />;
};

export default Home;
