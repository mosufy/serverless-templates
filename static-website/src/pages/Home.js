import React from "react";

import HomeHeader from "components/molecules/HomeHero";

const Home = () => {
  return <HomeHeader />;
};

export default Home;
