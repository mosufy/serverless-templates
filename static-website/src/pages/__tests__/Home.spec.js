import React from "react";
import { shallow } from "enzyme";

import Home from "../Home";

describe("test Home page", () => {
  it("should render correctly", () => {
    const component = shallow(<Home />);
    expect(component).toMatchSnapshot();
  });
});
