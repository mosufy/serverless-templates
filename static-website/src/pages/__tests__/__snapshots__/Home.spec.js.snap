// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Home page should render correctly 1`] = `<HomeHero />`;
